COPY/usr/bin/manageiq-db-ready

```diff
@@ -26,7 +28,12 @@ def manageiq_db_ready?(db_yaml, env = "production")
   host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
   password = ManageIQ::Password.try_decrypt(password)
 
-  results = `PGPASSWORD=#{password} psql --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
+  # If not present, let psql decide to use localhost, socket, etc.
+  host_arg = host.nil?     ? "" : "--host=#{host}"
+  port_arg = port.nil?     ? "" : "--port=#{port}"
+  pass_arg = password.nil? ? "" : "PGPASSWORD=#{password}"
+
+  results = `#{pass_arg} psql #{host_arg} #{port_arg} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
```

Comment:
Just because I like code golf ⛳ 

```suggestion
  # If not present, let psql decide to use localhost, socket, etc.
  host     &&= "--host=#{host}"
  port     &&= "--port=#{port}"
  password &&= "PGPASSWORD=#{password}"

  results = `#{password} psql #{host} #{port} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
```

Reply:
Ugh.  Really?  

So, now we're inserting nils that become whitespace at the front of the command? 🤮 

I was just trying to fix a bug.

Reply:
LOL no - I just was playing around 😉